src/api/utils/fs_utils.ts

```diff
@@ -31,16 +31,24 @@ export function rimraf(thing: string) {
 }
 
 export function copy(from: string, to: string) {
-	if (!fs.existsSync(from)) return;
-
-	const stats = fs.statSync(from);
-
-	if (stats.isDirectory()) {
+	const copied = new Set();
```

Comment:
Copy is used in a number of different places, but you don't need a `Set()` in any but the one. It would be better to leave the creation of the `Set` outside of copy.